### PR TITLE
docker and more

### PR DESCRIPTION
**Pull request by NullableInt**
🤔 Site generation takes a bit too long. Over 8 seconds we have to wait between saving a file before we can furiously press the refresh button.
No more! No more shall this stand!
I present to you: 1.2 seconds site generation AND live reload!
No more refresh, when the site is ready it will refresh by itself. As it should do.

### CODE REVIEW

**Comment by NullableInt** [comment]
To get the fancy features you'll have to use the new `docker-compose`  method to run.

File: README.md [hunk 1]
```diff
@@ -18,18 +18,26 @@ well as [Swedbank Pay Open Source Development Guidelines][dev-guidelines].
...
 3. To install the [Ruby Gems][gems] this web site requires, you first need to
    install [Bundler][bundler].
-4. Once Ruby and Bundler is in place, type `bundle install` inside the root
+4. Install [Graphviz][graphviz], this will require Java.
+5. Once Ruby ,Bundler, and Graphviz, is in place, type `bundle install` inside the root
    folder of this repository.
-5. Run `bundle exec jekyll serve` to start the website.
-6. Open `http://localhost:4000` in a browser.
-7. In Visual Studio Code, install the following plugins:
+6. Run `bundle exec jekyll serve` to start the website.
```
Comment (payex-dev) [inline README.md:38]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5169114)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
- Added 11
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5169114&bid=16969618)

File: README.md [hunk 2]
```diff
@@ -18,18 +18,26 @@ well as [Swedbank Pay Open Source Development Guidelines][dev-guidelines].
...
+7. Open `http://localhost:4000` in a browser.
```
Comment (payex-dev) [inline README.md:39]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5169114)

Comment (payex-dev) [inline README.md:39]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5169114)

File: README.md [hunk 3]
```diff
@@ -18,18 +18,26 @@ well as [Swedbank Pay Open Source Development Guidelines][dev-guidelines].
...
 
 To view this website, browse to [developer.swedbankpay.com][swp-dp]. If you'd
-like to host it locally on your computer, you need to do the following:
+like to host it locally on your computer, you have to options;
+[docker][docker], or [manual][manual-install] install.
 
+### Docker
+
+1. Install [docker][docker-install] for your operating system.
+2. Run `docker-compose up`
```
Comment (payex-dev) [inline README.md:27]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5169114)

File: README.md [hunk 4]
```diff
@@ -18,18 +18,26 @@ well as [Swedbank Pay Open Source Development Guidelines][dev-guidelines].
...
+
+### Manual install
 1. [Clone this repository][cloning].
 2. Jekyll is written in [Ruby][ruby], so you'll need to download and install
    that. If you're installing on Windows, choose setup with DevKit.
 3. To install the [Ruby Gems][gems] this web site requires, you first need to
    install [Bundler][bundler].
-4. Once Ruby and Bundler is in place, type `bundle install` inside the root
+4. Install [Graphviz][graphviz], this will require Java.
+5. Once Ruby ,Bundler, and Graphviz, is in place, type `bundle install` inside the root
```
Comment (payex-dev) [inline README.md:36]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5169114)

File: README.md [hunk 5]
```diff
@@ -18,18 +18,26 @@ well as [Swedbank Pay Open Source Development Guidelines][dev-guidelines].
...
    folder of this repository.
-5. Run `bundle exec jekyll serve` to start the website.
-6. Open `http://localhost:4000` in a browser.
-7. In Visual Studio Code, install the following plugins:
+6. Run `bundle exec jekyll serve` to start the website.
+7. Open `http://localhost:4000` in a browser.
+8. In Visual Studio Code, install the following plugins:
```
Comment (payex-dev) [inline README.md:40]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5169114)

File: README.md [hunk 6]
```diff
@@ -18,18 +18,26 @@ well as [Swedbank Pay Open Source Development Guidelines][dev-guidelines].
...
+2. Run `docker-compose up`
+
+### Manual install
 1. [Clone this repository][cloning].
 2. Jekyll is written in [Ruby][ruby], so you'll need to download and install
    that. If you're installing on Windows, choose setup with DevKit.
 3. To install the [Ruby Gems][gems] this web site requires, you first need to
    install [Bundler][bundler].
-4. Once Ruby and Bundler is in place, type `bundle install` inside the root
+4. Install [Graphviz][graphviz], this will require Java.
```
Comment (payex-dev) [inline README.md:35]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5169114)

Comment (payex-dev) [inline README.md:35]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5169114)

File: README.md [hunk 7]
```diff
@@ -18,18 +18,26 @@ well as [Swedbank Pay Open Source Development Guidelines][dev-guidelines].
 ## Usage
 
 To view this website, browse to [developer.swedbankpay.com][swp-dp]. If you'd
-like to host it locally on your computer, you need to do the following:
+like to host it locally on your computer, you have to options;
+[docker][docker], or [manual][manual-install] install.
 
+### Docker
+
+1. Install [docker][docker-install] for your operating system.
```
Comment (payex-dev) [inline README.md:26]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5169114)

File: README.md [hunk 8]
```diff
@@ -43,7 +51,7 @@ like to host it locally on your computer, you need to do the following:
    * `supperchong.pretty-json` to format selected JSON snippets in code
      examples.
    * `sissel.shopify-liquid` for syntax highlighting of [Liquid][liquid].
-8. Also in Visual Studio Code, [set up a ruler at 80 characters][vsc-ruler]
+9. Also in Visual Studio Code, [set up a ruler at 80 characters][vsc-ruler]
```
Comment (payex-dev) [inline README.md:54]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5169114)

File: README.md (hunk 1 above)
Comment (payex-dev) [inline README.md:38]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5169114)

**Pull request merged**